### ISSUE TITLE
Tile gaps

### ISSUE THREAD

**Issue by SoulyVEVO** [DDNetPP/maps#45]
generic unhookables have those gaps between them that look bad and can be easly improved

I see 2 options to fix that

Fill gaps with background tiles/freeze
![a](https://user-images.githubusercontent.com/61762251/110417769-669dcf00-8096-11eb-9396-e26f9c5b7653.png)

Edit the tiles image to fill the gaps (its still exmbedded so shouldnt affect file size that much)
![b](https://user-images.githubusercontent.com/61762251/110417863-a4025c80-8096-11eb-87de-c07b0ca3c596.png)

Or we can leave it as it is now (ugly)
![c](https://user-images.githubusercontent.com/61762251/110417947-cb592980-8096-11eb-9480-834d2047199b.png)

**Comment by fokkonaut** [comment]
Is it even visible without zooming in that much?

**Comment by SoulyVEVO** [comment]
@fokkonaut i can see those gaps pretty clear in game

**Comment by ChillerDragon** [comment]
Yes I am also annoyed by this. I started experimenting with corners from blmapchill_txt mapres.
![screenshot_2021-03-09_09-57-52](https://user-images.githubusercontent.com/20344300/110445235-32d69f80-80be-11eb-8b1c-be5c92fbf11a.png)

It is a bit of work and I am not sure if it is the cleanest solution but you can see that the freeze fills the corners on the outside but in the inside of the generic there is none.

Could do the same with the background if it works out well I might add it to the whole map.
